someip-controller/Isomeip_controller.h

```diff
@@ -24,6 +24,10 @@ namespace someip {
 using SomeIPMethod = std::function<simba::core::Result<
     std::pair<std::vector<uint8_t>, com::core::data::MessageCode>>(
     const std::vector<uint8_t> payload)>;
+using SomeIPEvent = std::function<simba::core::Result<
+    std::pair<std::vector<uint8_t>, com::core::data::MessageCode>>(
+    const std::vector<uint8_t> payload)>;
+
```

Comment:
To nie musi nic zwracać
```suggestion
using SomeIPEvent = std::function<void>(
    const std::vector<uint8_t> payload)>;

```